### DISCUSSION TITLE
InputVolumeMeters?

### DISCUSSION THREAD

**Discussion by tankqj** [StartAutomating/obs-powershell#211] [Ideas]
Sorry if someone already figured this out but I am not able to find out nor do I have the skill to code this.  Is there a away to use the Powershell Modules to get the InputVolumeMeters values of an input?  End goal is to run a script every so often to make sure the microphone is hearing something (InputVolumeMeters don't show 0 value).

**Comment by StartAutomating** [comment]
@tankqj short answer is: this is a good idea and also a bit of a tricky one.

The volume change events are _very_ high volume, and the way things are currently wired this can crash a console (which nobody wants).

Looking into it though, since I've recently figured out some ways to improve the performance of WebSockets in PowerShell.
